src/lib/modules/console/suggestions.ts

```diff
@@ -1,32 +1,69 @@
 let utils = require('../../utils/utils');
 
-class Suggestions {
-  constructor(embark, options) {
+type Events = {
+  on: any,
+  request: any
+}
+
+type EmbarkApi = {
+  events: Events,
+  registerAPICall: any
+}
+
+type ContractsManager = {
+  [key: string]: any
+}
+
+type Suggestion = {
+  value: string,
+  command_type: string,
+  description: string
+}
+
+type SuggestionsList = Array<Suggestion>;
```

Comment:
As per discussion on Gitter and https://stackoverflow.com/questions/37233735/typescript-interfaces-vs-types#37233777 I think we're slightly better off using `interface` here, except for `SuggestionsList`